src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -352,6 +352,336 @@
   </div>
 </div>
 
+{% if responseType equals "FULL_ADMISSION" %}
+{% if paymentOption is defined %}
```

Comment:
That is unnecessary, payment option has to be defined at all times.

Reply:
removed